packages/core/test/acceptance/directive_spec.ts

```diff
@@ -253,4 +253,244 @@ describe('directives', () => {
 
   });
 
+  describe('attribute ghosting behaviors', () => {
+    /**
+     * To match ViewEngine, we need to ensure the following behaviors
+     */
+
+    @Directive({
+      selector: '[dir-with-title]',
+    })
+    class DirWithTitle {
+      @Input()
+      title = ''
+    }
+
+    it('should set both the div attribute and the directive property for `title="value"`', () => {
+      @Component({template: `<div dir-with-title title="a"></div>`})
+      class App {
+      }
+
+      TestBed.configureTestingModule({
+        declarations: [App, DirWithTitle],
+      });
+      const fixture = TestBed.createComponent(App);
+      fixture.detectChanges();
+
+      const dirWithTitle =
+          fixture.debugElement.query(By.directive(DirWithTitle)).injector.get(DirWithTitle);
+      const div = fixture.nativeElement.querySelector('div');
+      expect(dirWithTitle.title).toBe('a');
+      expect(div.title).toBe('a');
```

Comment:
Nit: It's a bit confusing that the test says it's testing for an attribute on div, but we are actually looking at the property value on the div in the assertion. Functionally the value is the same here, but I think we should use attribute lookup to be precise:

```suggestion
      expect(div.getAttribute('title')).toBe('a');
```